### DISCUSSION TITLE
What does the setting for audio quality -1kbps mean?

### DISCUSSION THREAD

**Discussion by davidtaitabsa** [Shabinder/SpotiFlyer#486] [Q&A] [answered]
The is a new setting listed as -1 kbps under settings.  What does this setting do?

**Comment by Shabinder** [comment, marked as answer]
Oh that shouldnt be visible to user, Its a mishap and will be not present in coming updates, don't select that as it can cause unforeseen issues

**Comment by Marcelo698** [comment]
Seleccione 320kb y descargar lo contrario. 124kb
